test/test_crop.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+import numpy as np
+from lailib.image.crop import crop_boundary_and_padding
+
+class TestCropBoundaryAndPad:
```

Comment:
I don't do those checks because the res arrays(or res images) are uint8 gray scale arrays. If the output image is not uint8 gray scale image, the assertion can not be passed. We assume some of the code in the test must have the desired result.